path_action collision_checking_path
---
bool collision
